rtengine/iimage.h

```diff
@@ -98,6 +98,10 @@ class ImageDatas : virtual public ImageDimensions
         return "unknown";
     }
 
+    virtual bool isBW () const
```

Comment:
@Floessie I agree that LTO should be on by default. I also had favorable experiences using GCC and LTO in embedded projects. But I also had GCC fail horribly on me, actually especially using `-Os`, by removing stuff but actually needing it later. It seemed to depend very much on the GCC version at the time and hopefully everything is now fine.
I think a good start would be to ask package maintainers to enable it where recent GCC versions are used? For example we could ask @mbajor if he could enable it for his packages on OpenSUSE 42.1 when building with GCC 5.3? Just didn't find the time yet...